src/main/frontend/client/components/TrailShow.js

```diff
@@ -1,64 +1,82 @@
-import React, { useState, useEffect } from 'react';
-import { Link } from 'react-router-dom'
-
-const TrailShow = props => {
-  const [trail, setTrail] = useState({});
-  const trailId = props.match.params.id;
-  const {
-    name,
-    description,
-    distance,
-    elevationGain,
-    difficulty,
-    zipCode,
-    rating,
-    imgUrl
-  } = trail;
-
-  const getTrail = async () => {
-    try {
-      const res = await fetch(`/api/v1/trails/${trailId}`);
-      if (!res.ok) {
-        const error = new Error(`${res.status} (${res.statusText})`);
-        throw(error);
-      }
-      const trailData = await res.json();
-      setTrail(trailData.trail);
-    } catch (e) {
-      console.error("Error in fetch: ", e.message);
-    }
-  }
-
-  useEffect(() => {
-    getTrail();
-  }, []);
-
-  return (
-    <div>
-      <div>
-        <img src={imgUrl} />
-      </div>
-      <div>
-        <h1>{name}</h1>
-      </div>
-      <div className="grid-x grid-margin-x">
-        <div className="cell small-12 medium-4">
-          <p>Difficulty: {difficulty}</p>
-          <p>Distance: {distance} miles</p>
-          <p>Elevation Gain: {elevationGain} ft</p>
-          <p>Location: {zipCode}</p>
-          <Link
-            to={{ pathname: `/trails/${trailId}/edit`, state: { trail: trail } }}>
-            <button type="button" className="button">Edit Trail</button>
-          </Link>
-        </div>
-        <div className="cell small-12 medium-8">
-          <h3>Description</h3>
-          <p>{description}</p>
-        </div>
-      </div>
-    </div>
-  );
-};
-
+import React, { useState, useEffect } from 'react';
```

Comment:
This is really odd to me that about 60 lines of code are the same (or very similar?) but Git picked it up as a change.  Did the main coder change?  Your formatting settings may be different, and if so, it'll make seeing changes/differences harder as Git will assume you're just changing out the whole file and it'll be difficult to look for diffs.